aries_cloudagent/protocols/connections/v1_0/manager.py

```diff
@@ -1311,3 +1313,56 @@ async def update_inbound(
             if conn_info.verkey == recip_verkey:
                 connection.routing_state = routing_state
                 await connection.save(self._session)
+
+    async def set_transaction_my_role(
```

Comment:
Probably want to use a different term other than "role", and probably don't want to add these methods to the connection manager (since they are endorser protocol specific)

Reply:
Hi @ianco, Ok. We would move the entire code which sets the transaction related roles in a connection record to the endorse_transaction folder, so that it comes with the set of endorse_transaction APIs

Reply:
Hi @ianco, In the latest commit we have separated the code which sets the metadata and transferred it to the set of endorse-transaction APIs. Also, we have changed the name from roles to jobs wherever it is required. Can you check it?